foundations/foundation-config/src/main/java/org/apache/servicecomb/config/spi/ConfigCenterConfigurationSource.java

```diff
@@ -22,5 +22,6 @@
 import com.netflix.config.WatchedConfigurationSource;
 
 public interface ConfigCenterConfigurationSource extends WatchedConfigurationSource {
+  boolean isValidSource(Configuration localConfiguration);
```

Comment:
For compatibility, this method should give default implementation. But it's fine now , this is not known implementations.